frontend/src/scenes/billing/CurrentUsage.tsx

```diff
@@ -16,13 +16,39 @@ export function CurrentUsage(): JSX.Element | null {
         <>
             <div className="space-top" />
             <Card title="Current monthly usage">
+                {billing.should_display_current_bill && (
+                    <>
+                        <h3 className="l3">Current bill amount</h3>
+                        {billing?.current_bill_amount !== undefined && billing?.current_bill_amount !== null ? (
+                            <>
+                                This is the amount (in dollars) of the bill for the currently ongoing period. The final
+                                amount will be billed a few days after the end of the month. Please note this number is
+                                reported on a daily basis,{' '}
+                                <b>so events ingested in the last 24 hours may not be reflected yet.</b>
+                                <div className="bill-amount">
+                                    {`$${billing?.current_bill_amount?.toLocaleString()}`}
+                                </div>
+                            </>
+                        ) : (
+                            <>
+                                We can't show your current bill amount right now. Please check back again in a few
```

Comment:
I think here too it would be helpful to give a more descriptive timeline than checking back in a few minutes i.e. this is normal, please check back in 24 hours, then contact us

Reply:
I think a few minutes is accurate. The problem is not that the bill hasn't been calculated, if the period is just starting this will show as $0, not unavailable. Unavailable would only be shown if there's an exceptional problem contacting Stripe.